login.html

```diff
@@ -114,30 +114,31 @@
       submitForm();
      }
   }
-  function submitForm() {
+  function submitForm(postAction) { //Takes either login, register, or resend
```

Comment:
looks like this picked up the other commit, so let's land that PR first to keep this one simple